silo-core/contracts/interestRateModel/InterestRateModelV2.sol

```diff
@@ -197,8 +191,10 @@ contract InterestRateModelV2 is IInterestRateModel, IInterestRateModelV2 {
         Setup memory siloSetup = getSetup[_silo];
         fullConfig = irmConfig.getConfig();
 
-        fullConfig.ri = siloSetup.ri;
-        fullConfig.Tcrit = siloSetup.Tcrit;
+        if (siloSetup.initialized) {
+            fullConfig.ri = siloSetup.ri;
+            fullConfig.Tcrit = siloSetup.Tcrit;
+        } // else starting with original full setup
```

Comment:
```suggestion
        // if initialized, read ri and Tcrit from storage. Otherwise use config values.
        if (siloSetup.initialized) {
            fullConfig.ri = siloSetup.ri;
            fullConfig.Tcrit = siloSetup.Tcrit;
        }
```